cms/envs/common.py

```diff
@@ -465,36 +465,12 @@
         ],
         'output_filename': 'css/cms-style-app.css',
     },
-    'style-app-extend1': {
-        'source_filenames': [
-            'sass/style-app-extend1.css',
-        ],
-        'output_filename': 'css/cms-style-app-extend1.css',
-    },
     'style-app-rtl': {
         'source_filenames': [
             'sass/style-app-rtl.css',
         ],
         'output_filename': 'css/cms-style-app-rtl.css',
     },
-    'style-app-extend1-rtl': {
-        'source_filenames': [
-            'sass/style-app-extend1-rtl.css',
-        ],
-        'output_filename': 'css/cms-style-app-extend1-rtl.css',
-    },
-    'style-xmodule': {
```

Comment:
``style-xmodule`` was an extend we used for Studio (rather than ``*-extend2``). I remember naming this file this was because it really only dealt with the xmodule section of the app's compilation list. Those partials needed the variables and items loaded first.

I'm not messing with any xmodule clean up here, just removing a uniquely named extend.